persistence/sql/persister_errorx.go

```diff
@@ -30,6 +30,7 @@ func (p *Persister) Add(ctx context.Context, csrfToken string, errs ...error) (u
 	c := &errorx.ErrorContainer{
 		CSRFToken: csrfToken,
 		Errors:    buf.Bytes(),
+		SeenAt:    time.Now().UTC(),
```

Comment:
SeenAt should be set when the error has been read. I think we might have to use a `*time.Time` pointer to fix this.

Reply:
@aeneasr OK, then please close this PR and handle the issue by yourself.

Reply:
Ok, I will do that - thank you for your contribution :) Fixing this was actually quite a lot of work as I had to fix several issues in an upstream project for migrations to properly work as `seen_at` was actually NOT NULL (though it should be NULLable) 😅